var/spack/repos/builtin/packages/py-optax/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyOptax(PythonPackage):
+    """A gradient processing and optimisation library in JAX."""
+
+    homepage = "https://github.com/deepmind/optax"
+    pypi = "optax/optax-0.1.7.tar.gz"
+
+    license("Apache-2.0")
+
+    version("0.1.7", sha256="6a5a848bc5e55e619b187c749fdddc4a5443ea14be85cc769f995779865c110d")
```

Comment:
There is a newer version (0.2.11) available on pypi as well.

Reply:
Hello @manuelakuhn, your mentioned version refers to a different package, opax not optax

Reply:
ah sorry, then don't mind :-)